src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -290,7 +266,8 @@ class DefaultColorControlSceneHandler : public scenes::DefaultSceneHandlerImpl
                 if (decodePair.valueUnsigned8.Value() <=
                     static_cast<uint8_t>(ColorControl::EnhancedColorMode::kEnhancedCurrentHueAndCurrentSaturation))
                 {
-                    targetColorMode = decodePair.valueUnsigned8.Value();
+                    targetColorMode =
+                        static_cast<chip::app::Clusters::ColorControl::EnhancedColorModeEnum>(decodePair.valueUnsigned8.Value());
```

Comment:
```suggestion
                        static_cast<ColorControl::EnhancedColorModeEnum>(decodePair.valueUnsigned8.Value());
```
Please stop with all the extra prefixes.  I've said that a number of times on this PR already....